data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,377 @@
+			choice
+				`	"Then let me help them."`
+					goto initiative
+				`	"You can deal with the consequences later. Help them now!"`
+			`	The Quarg takes on a sour look. "Insisting you are. Understand that our own intervention is out of the question." It sighs. "However, I can foresee the likelihood of you making a rash and impetuous decision to contact this ship."`
+			choice
+				`	"Sure, let me do the work."`
+				`	"If you are too weak..."`
+			`	The Quarg rapidly clicks its teeth, before continuing.`
+			label initiative
+			`	"You lack the tools for such a mission. To my knowledge, no vessels capable of landing on gas giants have ever been developed by your kind. And you are certainly not able to make your intentions known to them in any meaningful sense."`
+			`	"But for the sake of the Incipias, I will assist you on this last point. If you cannot be dissuaded from your course of action, then it is best you are given a means of communication, so that your interactions can at least be on amicable terms. I can provide you with a single one of our translators, which will allow you to contact the species in their home system. Request that they sell you one of their ships, so that you are able to land on their planets."`
+			`	It pauses before taking on a more serious tone. "However, you must remember one thing. You are forbidden from granting them any advanced technology of yours. If you agree to that point we shall assist you."`
+			label choice
+			choice
+				`	"If that's how I can help them, sure."`
+					goto yes
+				`	"Why do you not want me to sell my tech?"`
+			`	"Halting the natural increase of a species is unpredictable at the best of times. Your species lacks the wisdom to understand this, as you are young, and have not seen what we have seen." The Quarg pauses for a moment, lost in thought. "The foreseeable risks of even letting you contact them are monumental, and not something you should consider trivial."`
+				goto choice
+			label yes
+			`	"Happy I am that you agree to that condition. Bear in mind, Captain, that the slightest misstep may have grave consequences." The Quarg stares you down. "We will be watching your actions, to make sure that they are peaceful and without ill or unwise intent. Please for a moment wait here," it says, and walks away.`
+			`	The Quarg only takes a moment to come back; it is now carrying a sleek silver box with a display on one side.`
+			`	"This is our translator, our link to the Incipias. It is able to translate into your language as well. Be aware of the honor of me handing this to you," it says.`
+			`	As you stow it away, the Quarg speaks once more. "To satisfactorily utilize this device, you must point it towards the source of the communication, so that it is able to pick up on the emission. Given the complexity of their language, the translator might not be able to adequately translate some parts of their speech. You will understand it when it happens."`
```

Comment:
```suggestion
			`	As you stow it away, the Quarg speaks once more. "To satisfactorily utilize this device, you must direct it towards the source of the communication, so that it is made able to discern the relevant emissions. Given the complexity of their language, the translator might not be able to adequately translate some aspects of their obscure speech."`
```